src/app/upload/page.test.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import { render, screen, waitFor } from '@testing-library/react';
+
+import Upload from './page';
+
+describe('Upload', () => {
+  it('renders component', async () => {
```

Comment:
Not correct English:

```suggestion
  it('renders the component', async () => {
```